.env.example

```diff
@@ -53,7 +53,7 @@ ENABLE_DEFAULT_SCHEMA=1
 
 # String template that will be interpolated on the client using two route params,
 # repositoryId and activityId.
-PREVIEW_URL=https://myserver.com/repository/{repositoryId}/activity/{activityId}
+PREVIEW_URL=https://myserver.com/api/v1/preview/
```

Comment:
How about: `http://boutique.localhost/api/v1/preview` so reader knows it points to boutique-esque LMS, additionally comment above does not longer apply :bowtie: